Dockerfile

```diff
@@ -1,11 +1,38 @@
-FROM degica/rails-base:2.5
+FROM degica/rails-buildpack:2.6 AS builder
 
-RUN apt-get update && apt-get install -y openssh-client --no-install-recommends && rm -rf /var/lib/apt/lists/*
+ENV APP_HOME=/app
+WORKDIR $APP_HOME
+
+COPY Gemfile $APP_HOME/
+COPY Gemfile.lock $APP_HOME/
+
+RUN bundle install -j=4 --without development test
+
+FROM ruby:2.6.1-slim
+
+ENV APP_HOME=/app
+ENV PATH=$APP_HOME/bin:$PATH
+WORKDIR $APP_HOME
+
+RUN useradd --user-group --create-home app
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+      file \
+      git \
+      curl \
+      libpq5 \
+      libsqlite3-0 \
+      libxslt1.1 \
+      libxml2 \
+      libcurl3 \
+      openssh-client \
+ && rm -rf /var/lib/apt/lists/*
+
+COPY --from=builder /usr/local/bundle /usr/local/bundle
+ADD --chown=app:app . $APP_HOME
 
-ADD . $APP_HOME
-RUN chown -R app:app $APP_HOME
 USER app
-RUN rake assets:precompile
```

Comment:
Where and when are assets compiled?

Reply:
Well I've noticed Barcelona is API only and doesn't have any assets :sweat_smile: so I just removed this line

Reply:
Ah I see